tgui/packages/tgui/layouts/Window.tsx

```diff
@@ -48,7 +49,8 @@ export const Window = (props: Props) => {
     height,
   } = props;
 
-  const { config, suspended, debug } = useBackend();
+  const { config, suspended } = useBackend();
+  const { debugLayout = false } = useDebug();
   if (suspended) {
     return null;
   }
```

Comment:
FYI this code is subtly broken because hooks can't be called conditionally. In this case, `useEffect` is running after a conditional return.

You need to move this after, and then *also* check inside the useEffect (and then add `suspended` as a dependency)